oregon/templates/home.html

```diff
@@ -90,4 +42,10 @@
   </div><!-- end summary -->
   <div class="clear"><!-- --></div>
 {% endblock %}
- 
+{% block javascript %} {{ block.super }}
+    <script src="http://d3js.org/d3.v3.min.js"></script>
+    <script src="http://d3js.org/topojson.v1.min.js"></script>
```

Comment:
Let's get this code locally unless there's some specific reason not to.  When we switch to https, this will break without an https endpoint on their side.  If they switch to https before us, we'll have to fix the code to keep linking.  If they do a minor update that breaks anything we have, we'll have no idea what happened an no way to revert.